src/pages/api/sentry-example-api.js

```
@@ -0,0 +1,5 @@
+// A faulty API route to test Sentry's error monitoring
+export default function handler(_req, res) {
+  throw new Error('Sentry Example API Route Error')
+  res.status(200).json({ name: 'John Doe' })
```

Biome

[error] This code is unreachable

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)